x-pack/plugins/security_solution/public/alerts/pages/detection_engine/rules/create/helpers.test.ts

```diff
@@ -481,13 +499,18 @@ describe('helpers', () => {
       };
       const result: AboutStepRuleJson = formatAboutStepData(mockStepData);
       const expected = {
```

Comment:
Can we type this with the Schema at this point?

```ts
const expected: RuleSchema
```

Not a biggie but would a nice win to know that we have more compile time checks when we change things.